dashboard/utils_test.py

```diff
@@ -26,21 +25,21 @@ class MMTrackTest(TestCase):
     Tests for the MMTrack class
     """
 
+    # create Enrollments, Certificates, CurrentGrades
+    enrollments_json = load_json_from_file('dashboard/fixtures/user_enrollments.json')
+    certificates_json = load_json_from_file('dashboard/fixtures/certificates.json')
+    current_grades_json = load_json_from_file('dashboard/fixtures/current_grades.json')
+
     @classmethod
     def setUpTestData(cls):
         super().setUpTestData()
         # create an user
         cls.user = UserFactory.create()
 
-        # create Enrollments, Certificates, CurrentGrades
-        enrollments_json = load_json_from_file('dashboard/fixtures/user_enrollments.json')
-        cls.enrollments = Enrollments(enrollments_json)
-
-        certificates_json = load_json_from_file('dashboard/fixtures/certificates.json')
-        cls.certificates = Certificates([Certificate(cert_json) for cert_json in certificates_json])
-
-        current_grades_json = load_json_from_file('dashboard/fixtures/current_grades.json')
-        cls.current_grades = CurrentGrades([CurrentGrade(grade_json) for grade_json in current_grades_json])
+        with patch('dashboard.models.CachedEnrollment.data_qset', return_value=cls.enrollments_json), \
+                patch('dashboard.models.CachedCertificate.data_qset', return_value=cls.certificates_json), \
+                patch('dashboard.models.CachedCurrentGrade.data_qset', return_value=cls.current_grades_json):
```

Comment:
this is inaccurate, given that the `data_qset` returns a query-set and not a list